### PR TITLE
4.0.0-alpha.0

### PR DESCRIPTION
**Pull request by Hoten**
bump!

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ echo "Test the lighthouse-viewer build"
 # Start a server in dist/viewer/ and open the page in a tab. You should see the viewer.
```
[the remark this answers is not in the document]
Comment (Hoten) [reply docs/releasing.md]: I assumed it was necessary ... I tried `python -m SimpleHTTPServer` first but I interpreted the viewer2x 400 error by thinking I need to use the static-server.

wdyt of instead suggesting `cd dist/viewer ; python -m SimpleHTTPServer`

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,144 @@
...
+* proto: make NO_ERROR default in proto ([#6358](https://github.com/googlechrome/lighthouse/pull/6358))
+* proto: reduce configSettings in proto to minimum ([88b2829](https://github.com/googlechrome/lighthouse/commit/88b2829))
+* yarn: update lockfile for micromatch ([#6384](https://github.com/googlechrome/lighthouse/pull/6384))
+
+## New Contributors!
+Thanks to @carlesnunez and @justin0022 for their first contributions!
+
+Thanks to @CodeDem for improving how the image-aspect-ratio audit handles small images!
+
+Huge thanks to myself (@Hoten) for joining Lighthouse! Go me!
```
Comment (patrickhulce) [inline changelog.md]: 😆

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,144 @@
+ <a name="4.0.0-alpha.0"></a>
+# 4.0.0-alpha.0 (2018-10-31)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v3.2.0...v4.0.0-alpha.0)
```
[the remark this answers is not in the document]
Comment (Hoten) [reply changelog.md]: should we just leave it as 3.2.0 ? with or without noting the technicality?

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,142 @@
+ <a name="4.0.0-alpha.0"></a>
+# 4.0.0-alpha.0 (2018-10-31)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v3.2.1...v4.0.0-alpha.0)
+
+## New Contributors!
+Thanks to @carlesnunez and @justin0022 for their first contributions!
+
+Thanks to @CodeDem for improving how the image-aspect-ratio audit handles small images!
+
+Huge thanks to myself (@Hoten) for joining Lighthouse! Go me!
```
Comment (patrickhulce) [inline changelog.md]: 🤣

File: changelog.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (brendankenny) [reply changelog.md]: I think it's totally fine to ignore the technicality and leave as 3.2.0. If someone's very meticulous record keeping is thrown off, we'll just take the heat :)

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,142 @@
+ <a name="4.0.0-alpha.0"></a>
+# 4.0.0-alpha.0 (2018-10-31)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v3.2.0...v4.0.0-alpha.0)
+
+## New Contributors!
+Thanks to @carlesnunez and @justin0022 for their first contributions!
+
+Thanks to @CodeDem for improving how the image-aspect-ratio audit handles small images!
+
+Huge thanks to myself (@Hoten) for joining Lighthouse! Go me!
```
Comment (exterkamp) [inline changelog.md]: Back in my day we just let @paulirish release, and he never mentioned if you even existed. :eyes:

File: changelog.md [hunk 6]
```diff
@@ -1,3 +1,142 @@
...
+* bail on run if insecure ssl certification ([#6300](https://github.com/googlechrome/lighthouse/pull/6300))
+* make throttled offscreen-images work without TTI ([#6039](https://github.com/googlechrome/lighthouse/pull/6039))
+* save HTMLElement.p.matches function to avoid conflict ([#6283](https://github.com/googlechrome/lighthouse/pull/6283))
+* remove fonts gatherer, move font-display to use CSSUsage artifact ([#6166](https://github.com/googlechrome/lighthouse/pull/6166))
+* remove last dependency on WebInspector ([#6209](https://github.com/googlechrome/lighthouse/pull/6209))
+* add timeout to all protocol commands ([#6347](https://github.com/googlechrome/lighthouse/pull/6347))
+* don't load blank page twice in first pass ([#6369](https://github.com/googlechrome/lighthouse/pull/6369))
+* refactor finding of trace start ([#6370](https://github.com/googlechrome/lighthouse/pull/6370))
+* increase Page.navigate timeout ([#6412](https://github.com/googlechrome/lighthouse/pull/6412))
+
```
Comment (exterkamp) [inline changelog.md]:
```suggestion
* add LH.Result object support to prepareLabData ([#6432](https://github.com/GoogleChrome/lighthouse/pull/6432))
```

File: lighthouse-viewer/app/src/lighthouse-report-viewer.js [hunk 7]
```diff
@@ -138,11 +138,13 @@ class LighthouseReportViewer {
   _replaceReportHtml(json) {
     this._validateReportJson(json);
 
-    if (!json.lighthouseVersion.startsWith('3')) {
+    if (json.lighthouseVersion.startsWith('2')) {
```
Comment (exterkamp) [inline lighthouse-viewer/app/src/lighthouse-report-viewer.js]: Not sure about this section, but do we need to worry that the 3x viewer now won't render a 4x LHR (b/c the crc children removal?)  Is that what line 146-147 are?

File: docs/releasing.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulirish) [reply docs/releasing.md]: wfm!

File: lighthouse-viewer/app/src/lighthouse-report-viewer.js (hunk 7 above)
Comment (paulirish) [reply lighthouse-viewer/app/src/lighthouse-report-viewer.js]: when we deploy-viewer that'll update the _default_ viewer which should handle 3x and 4x fine (incl crc). That's my running expectation.

@brendankenny will have to be smart about choosing to use to the PWACategoryRenderer

File: changelog.md (hunk 6 above)
Comment (Hoten) [reply changelog.md]: OK, I'll add this just before releasing (so I don't gotta keep merging master + updating the changelog)

**Comment by Hoten** [comment]
Travis is failing, not sure why.

File: changelog.md (hunk 6 above)
Comment (paulirish) [reply changelog.md]: it's fine to leave this particular commit out of the alpha.

**Pull request merged**